e2e/tests/desktop/registered-shopper.spec.js

```diff
@@ -165,3 +167,44 @@ test("Registered shopper can add item to wishlist", async ({ page }) => {
   // wishlist
   await validateWishlist({page})
 });
+
+/**
+ * Test that social login persists a user's shopping cart
+ */
+test("Registered shopper logged in through social retains persisted cart", async ({ page }) => {
```

Comment:
Will the basket persist for this user over time. For example will the e2e run one day to the next and accumulate items in that cart until it potentially errors out?

We could possibly remove the item from the cart after checking for its existence to be safe.

Reply:
From my testing, the basket doesn't persist!